pom.xml

```
@@ -1,12 +1,63 @@
 <?xml version="1.0" encoding="UTF-8"?>
+
+
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-
+<!-- cljkasdnckljansdc -->
     <groupId>com.schoolman</groupId>
     <artifactId>skyline</artifactId>
     <version>1.0.0.0-SNAPSHOT</version>
+    <packaging>war</packaging>
 
-    
+    <name>skyline</name>
+    <url>http://maven.apache.org</url>
+    <properties>
+        <spring.version>4.3.8.RELEASE</spring.version>
```

Is this being used anywhere in the pom ?

---

No I used it only once

---

Where did you use it ? i dont see it in the pom anywhere ? This project doesnt even seem to have spring dependency

---

I deleted it and checked. The changes are working fine
```suggestion
        <spring.version>4.3.8.RELEASE</spring.version>
```